From ffa12e505896be05aa35188747e77370cff279ba Mon Sep 17 00:00:00 2001
From: Timothy Johansson <timothy.ejohansson@gmail.com>
Date: Tue, 12 Apr 2011 21:26:35 +0200
Subject: [PATCH 2/2] Some minor changes

---
 README |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/README b/README
index b46bac4..fad8e72 100644
--- a/README
+++ b/README
@@ -1,6 +1,6 @@
 Shortfin Web Server
 ------------------
-  by Timothy E. Johansson, 2011
+ by Timothy E. Johansson, 2011
 
 Description:
   Shortfin is a fast and lightweight static web server.
@@ -9,7 +9,7 @@ Description:
 Tested on:
 -  Linux (Ubuntu)
   
-
+  
 Install
 	# ./configure
  	# make
-- 
1.7.1

